x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/endgame/helpers.ts

```diff
@@ -40,14 +40,14 @@ export const getHumanReadableLogonType = (endgameLogonType: number | null | unde
 export const getHostNameSeparator = (eventAction: string | null | undefined): string =>
   eventAction === 'explicit_user_logon' ? i18n.TO : '@';
 
-export const useTargetUserAndTargetDomain = (eventAction: string | null | undefined): boolean =>
```

Comment:
@andrew-goldstein seems only hooks can start with `use`, are you ok to change the name to `get` or maybe do you have some other suggestion for the name?